lib/mri/neo4j/driver/session.rb

```
@@ -60,13 +60,14 @@ def run(query, parameters = {}, config = {})
         # `imp_user` (Bolt 4.4+) makes the server run the query as if
         # the named user had issued it — auth as the session's user,
         # authz as the impersonated one.
+        bookmarks = current_bookmarks_for_extra
         run_extra = {
-          db: @options[:database],
+          db: operation_database(bookmarks),
           mode: (session_access_mode == :read ? 'r' : nil),
           tx_timeout: timeout_to_milliseconds(timeout),
           tx_metadata:,
```

RuboCop (1.87.0)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)